### PR TITLE
Add freifunk-meet app

### PR DESCRIPTION
**Pull request by awlx**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [ ] `brew cask audit --download {{cask_file}}` is error-free.
=> Contains a warning because we use latest.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/freifunk-meet.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cask 'freifunk-meet' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://github.com/FreifunkMUC/jitsi-meet-electron/releases/latest/download/ffmuc-meet.dmg'
```
Comment (miccal) [inline Casks/freifunk-meet.rb:5]: Please change this to:
```
url "https://github.com/freifunkMUC/jitsi-meet-electron/releases/download/v#{version}/ffmuc-meet.dmg"
```

File: Casks/freifunk-meet.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+cask 'freifunk-meet' do
+  version :latest
+  sha256 :no_check
```
Comment (miccal) [inline Casks/freifunk-meet.rb:3]: Pleas add the latest `version` and `sha256` information.

**Comment by ran-dall** [comment]
Welcome to HBC and thanks for your first contribution. Unfortunately, [this does not meet HBC notability requirements at the current time](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/rejected_casks.md).

> The Cask is for an app that is too obscure. Examples:
> - A self-submitted app from a GitHub repository that is not notable enough (under 30 forks, 30 watchers, and 75 stars).
> - Electronic Identification (eID) software.

Also, we have a special tap for forks...
> We maintain separate Taps for different types of binaries. Our nomenclature is:
> ...
> Fork: An alternate version of an existing project, with a based-on but modified source and binary.
 
That said, you're also more than welcome to host this on your own personal tap. Thanks again for submitting!

**Comment by awlx** [comment]
As we provide this app for endusers, there are no forks or stars. Would proof of "enough" downloads be enough to make it to the forks tap?
```
bash-3.2$ curl -s https://api.github.com/repos/freifunkMUC/jitsi-meet-electron/releases/tags/v2.0.17 | jq '.assets[] | "\(.name) \(.download_count)"'
"ffmuc-meet-amd64.deb 9"
"ffmuc-meet-x86_64.AppImage 168"
"ffmuc-meet.dmg 62"
"ffmuc-meet.dmg.blockmap 1"
"ffmuc-meet.exe 788"
"ffmuc-meet.exe.blockmap 578"
"ffmuc-meet.zip 140"
"latest-linux.yml 222"
"latest-mac.yml 406"
"latest.yml 1120"
```

**Comment by ran-dall** [comment]
@awix I don't know the specifics to that tap. I was referring to your app, it shows it is a fork. `forked from jitsi/jitsi-meet-electron`
Screen Shot 2020-06-21 at 2 53 04 PM

**Comment by awlx** [comment]
Yep, it's a fork to make it easier for users to connect to the german Servers. But my comment was more a reference to the "The Cask is for an app that is too obscure.".

How to provide a proof that this might be relevant for enough users? :)

But thank you for your in detail explanation and your time :).

**Comment by simphide** [comment]
push

**Pull request closed without merging**